src/devnet/venv-devnet.ts

```diff
@@ -0,0 +1,24 @@
+import { ChildProcess, spawn } from "child_process";
+
+import { getPrefixedCommand, normalizeVenvPath } from "../utils/venv";
+import { IntegratedDevnet } from "./integrated-devnet";
+
+export class VenvDevnet extends IntegratedDevnet {
+    private command = "starknet-devnet";
```

Comment:
I find it a bit confusing that `command` gets defined here only to potentially get redefined later in the constructor.

Reply:
[04a7e80](https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/77/commits/04a7e80f97c284b96151c70bd219d20eb6f6608f)